### DISCUSSION TITLE
【第四期飞桨黑客松】生态伙伴开源贡献——Arm项目

### DISCUSSION THREAD

**Discussion by cloud2009** [PaddlePaddle/Paddle#50736] [General]
[PaddlePaddle Hackathon](https://github.com/PaddlePaddle/Paddle/issues/50629#arm)
大家可以在这里讨论有关Arm项目任务的问题哦～

**Comment by PommesPeter** [comment]
请问 RFC 有模板么

**Reply by Liliyaw** [reply]
Hi 同学, 欢迎关注黑客松Arm项目题目。我们特别的RFC文档模版哈，可以参考组委会提供的模版，也可以自己撰写。关键点在于写清楚以下几个要点即可:
1. 项目名称
2. 项目内容描述
3. 业内方案调研
简要谈谈对于Arm虚拟硬件的了解
4. 设计思路与实现方案(关键步骤): 
可以参考题目描述, 但是描述清楚所选择的Paddle模型(若已有模型请提供链接)以及原因
5. 预期结果
6. 项目开发时间计划表
 
后续有需要调整的地方我们会联系你更改的哈～欢迎积极报名参加～

**Reply by PommesPeter** [reply]
好的 谢谢
